tools/benchmark/cmd/watch.go

```diff
@@ -194,7 +194,7 @@ func doWatch(stream v3.Watcher, requests <-chan string) {
 
 func recvWatchChan(wch v3.WatchChan) {
 	for range wch {
-		if atomic.LoadInt32(&nrRecvCompleted) == int32(eventsTotal) {
+		if atomic.LoadInt32(&nrRecvCompleted) >= int32(eventsTotal) {
```

Comment:
Should this be allowed? By default there's a collision when setting `watched` so `eventsTotal` will under count. Maybe `eventsTotal` should account for collisions instead of being a lower bound?